text_collector_examples/smartmon.sh

```diff
@@ -73,7 +73,7 @@ parse_smartctl_info() {
   local disk="$1" disk_type="$2"
   while read line ; do
     info_type="$(echo "${line}" | cut -f1 -d: | tr ' ' '_')"
-    info_value="$(echo "${line}" | cut -f2- -d: | sed 's/^ \+//g')"
+    info_value="$(echo "${line}" | cut -f2- -d: | sed 's/^ \+//g' | sed 's/"/\\"/')"
```

Comment:
This could be simplified to `sed `s/^ \+//g ; s/"/\\/'`.

But, I also think that this might need a `g` on the second regexp to match all double quotes in the string.